speak/face.py

```diff
@@ -41,9 +40,10 @@
 
 class Status:
     def __init__(self):
+        self.speech=Speech.SpeechManager()
         self.voice = voice.defaultVoice()
-        self.pitch = espeak.PITCH_MAX / 2
-        self.rate = espeak.RATE_MAX / 2
+        self.pitch = self.speech.MAX_PITCH
+        self.rate = self.speech.MAX_RATE
 
```

Comment:
You should use self.speech.get_pitch() and self.speech.get_rate(), to get the values configured by the user instead of the max values. In the end, I should verify if speech is needed at all in the Status class

Reply:
Just made the changes.Would you recommend squashing the commits or is it fine as it is?

Reply:
Just made the changes.Would you recommend squashing the commits or is it fine as it is?

Reply:
Squash is better. Other can provide more feedback, and test the code